pip_chill/pip_chill.py

```diff
@@ -55,7 +55,7 @@ def chill(show_all=False):
     if show_all:
         ignored_packages = ()
     else:
-        ignored_packages = {"pip", "wheel", "setuptools", "pkg-resources"}
+        ignored_packages = {"pip", "wheel", "setuptools", "pkg-resources", "pip-chill"}
```

Comment:
this change requires updates in the UT's too

Reply:
I think both methods work as intended to omit pip-chill from the list. If the goal is to be used to create a requirements.txt file and use 

```shell
pip-chill > requirements.txt
``` 

This to me keeps it simpler than having to do 

```shell
pip-chill --omit-self > requirements.txt
```

The `--show-all` is still available to show all the packages.

Reply:
either way is ok of course => it just looked by design to be included in the end its @rbanffy 's call :-) 
I can help fixing the build or rework my already green branch to not show it by default

Reply:
Works for me. I believe a solution to exclude the pip-chill package keeps in line with the initial statement "Make requirements with only the packages you need". We don't need pip-chill when creating a requirements file for sharing packages with others and keeping only what we need to install.

Reply:
suggestion to close with the --no-chill option added

Reply:
works for me!